### PR TITLE
Missing inEarlyPane assertion

### PR DESCRIPTION
**Pull request by mkuthan**
Just for better alignment with Beam I added missing inEarlyPane assertion.

### CODE REVIEW

File: scio-test/src/main/scala/com/spotify/scio/testing/SCollectionMatchers.scala [hunk 1]
```diff
@@ -302,6 +302,20 @@ trait SCollectionMatchers extends EqInstances {
         value.matcher(_.inLatePane(window))
     }
 
+  /**
+   * SCollection assertion only applied to the specified window, running the checker only on the
+   * early pane for each key.
+   */
+  def inEarlyPane[T: ClassTag, B: ClassTag](
```
Comment (regadas) [inline scio-test/src/main/scala/com/spotify/scio/testing/SCollectionMatchers.scala:309]: I believe we can simplify this with following:
```suggestion
  def inEarlyPane[T](
```

**Review by regadas — COMMENTED** [review]
Thanks for this @mkuthan

Would it be possible to add a test for it in `SCollectionMatchersTest`?

**Comment by mkuthan** [comment]
> Thanks for this @mkuthan
> 
> Would it be possible to add a test for it in `SCollectionMatchersTest`?

@regadas - Scio just delegates to Beam, so it would be rather Beam test for early panes :)

But let me find some spare time for writing the test (perhaps for other pane related assertions as well). I’m starting a one week winter break on Friday so it could be delayed a bit.

**Comment by regadas** [comment]
@mkuthan yeah the idea is not really to test the logic 😄 it was really to just validate the type inference, the same way we do for the others. Anyway is not really a must-have.

**Review by regadas — APPROVED** [review]

**Comment by mkuthan** [comment]
> @mkuthan yeah the idea is not really to test the logic 😄 it was really to just validate the type inference, the same way we do for the others. Anyway is not really a must-have.

Thanks! I totally forgot about this PR :( tough week at work after week on skis

**Pull request merged**